src/issuer/issuer.service.ts

```diff
@@ -198,12 +198,29 @@ export class IssuerService {
      * Makes a call to the agent to create a credential definition
      */
     public async createCredDef(schema_id: string, tag: string, support_revocation: boolean): Promise<any> {
-        const data = {
-            schema_id,
-            tag,
-            support_revocation,
-        };
-        return await this.agentCaller.callAgent(process.env.AGENT_ID, process.env.ADMIN_API_KEY, 'POST', 'credential-definitions', null, data);
+
+        if (support_revocation) {
+
+            const registry_size = 200;
```

Comment:
It's best not to have hard coded values, we should use env variables instead. The question we need to ask is whether we think registry size will be constant across all our issuers, in which case we should set it here:
https://github.com/kiva/aries-guardianship-agency/blob/master/src/config/env.json#L2
Or whether we might want it to be specific for each issuer, in which case we should pass it in via the API. Since we already defined a value here:
https://github.com/kiva/protocol-aries/blob/master/implementations/kiva/profiles/employee.cred.def.json#L9
I think the best thing would be to add in a value passed through the API

Also I noticed we have 100 in the cred def and 200 here, which ever value we choose let's be consistent